com.github.simonwhitaker.Gibo.appdata.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="console-application">
+  <id>com.github.simonwhitaker.Gibo</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Unlicense</project_license>
```

Comment:
Is the maintainer aware of this?

Reply:
Yes, https://github.com/simonwhitaker/gibo/blob/master/UNLICENSE